firstvoices/backend/tests/test_models/test_page_model.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from django.db import IntegrityError
+
+from backend.tests import factories
+
+
+class TestSitePageModel:
+    @pytest.mark.django_db
+    def test_one_or_zero_banner(self):
+        image = factories.ImageFactory.create()
+        video = factories.VideoFactory.create()
+
+        try:
+            factories.SitePageFactory.create()
+            factories.SitePageFactory.create(banner_image=image)
+            factories.SitePageFactory.create(banner_video=video)
+        except IntegrityError:
```

Comment:
Could you adjust this a little bit to be sure the error is coming when you try to add the second banner and not before?